pkg/cmd/describe/domains/domains.go

```diff
@@ -0,0 +1,109 @@
+package domains
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/MaxwelMazur/tablecli"
+	msg "github.com/aziontech/azion-cli/messages/describe/domains"
+	"github.com/fatih/color"
+
+	api "github.com/aziontech/azion-cli/pkg/api/domains"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	var domainID string
+	opts := &contracts.DescribeOptions{}
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azion describe domains --domain-id 4312
+        $ azion describe domains --domain-id 1337 --out "./tmp/test.json" --format json
+        $ azion describe domains --domain-id 1337 --format json
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Why not define this function elsewhere and then, reference it in here?

Reply:
We've already asked ourselves that! And what we see is that all the other CLI players follow the same line, so we prefer to leave it that way. And this is also something from cobra itself Examples, https://github.com/docker/cli/blob/master/cli/command/plugin/inspect.go, https://github.com/cli/cli/blob/trunk/pkg/cmd/gist/create/create.go

Reply:
Very Ugly!

Reply:
It would be great to receive suggestions for improvements @karane